src/Games/NexusMods.Games.BethesdaGameStudios/ABethesdaGame.cs

```diff
@@ -31,6 +32,9 @@ protected ABethesdaGame(IServiceProvider provider) : base(provider)
             FomodXmlInstaller.Create(provider, new GamePath(LocationId.Game, "Data".ToRelativePath())),
             // Handles common installs to the game folder and other common directories like `Data`
             GenericFolderMatchInstaller.Create(provider, BethesdaInstallFolderTargets.InstallFolderTargets()),
+
+            // Handles everything else
+            // AdvancedInstaller<UnsupportedModOverlayViewModelFactory, AdvancedInstallerOverlayViewModelFactory>.Create(provider),
```

Comment:
Is this a left-over?

Reply:
That is where you would put it to test/use it. If I leave it there though it breaks tests because it requires the OverlayManager, that isn't available in CLI setting.